### PR TITLE
refactor(gh-pages): add missing ESLPanelGroup component and improve meta information

### PR DESCRIPTION
**Pull request by ala-n**
Demo sIte content fixes.

### CODE REVIEW

**Review by julia-murashko — APPROVED** [review]

File: pages/views/components/esl-alert.njk [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: content
 title: ESL Alert
```
Comment (dshovchko) [inline pages/views/components/esl-alert.njk:3]: As I understand it, the result will be:

ESL | ESL Alert

This is a very bad title! Unintelligible not only to the search engine but also to the potential visitor to the site. Why would he go to such a page?

The title should be something like this:

ESL Alert - a web component to show notifications on your pages

This is just a proposal.

File: pages/views/components/esl-alert.njk [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+description: Custom element to show small notifications on the page.
```
Comment (dshovchko) [inline pages/views/components/esl-alert.njk:4]: That's exactly the title content. And here as it seems to me should be a brief description of what the page contains. I.e. something like this:

An explanation of the ESL Alert web component and its possible uses, as well as examples of how to use it.

**Review by dshovchko — APPROVED** [review]

**Review by Anna-MariiaPetryk — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

File: pages/views/components/esl-panel-group.njk [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+layout: content
+title: ESL Panel Group
+seoTitle: ESL Panel Group - custom element to group ESLPanel-s to have Tabs or Accordion behaviour
```
Comment (fshovchko) [inline pages/views/components/esl-panel-group.njk:4]: "to have Tabs" is more correctly written like "and to have ESLTabs" or "and to give ESLTabs"

File: pages/views/components/esl-panel.njk [hunk 4]
```diff
@@ -1,11 +1,13 @@
 ---
 layout: content
 title: ESL Panel
+seoTitle: ESL Panel - custom element to collapse / expand content on the page (accordion behavior)
```
Comment (fshovchko) [inline pages/views/components/esl-panel.njk:4]: collapse / expand have whitespaces around "/" symbol

File: pages/views/components/esl-toggleable.njk [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: content
 title: ESL Toggleable
+seoTitle: ESL Toggleable - custom element to have basic show/hide or other state-full behaviour
```
Comment (fshovchko) [inline pages/views/components/esl-toggleable.njk:4]: show/hide doesn't have whitespaces around "/" symbol

File: pages/views/examples/footnotes.njk [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: content
 title: Footnotes
+seoTitle: Auto-collected footnotes custom elements example based on ESL web components
```
Comment (fshovchko) [inline pages/views/examples/footnotes.njk:4]: Based on other seoTitles this particular title should be rewritten like: "Custom auto-collected footnotes element examples based on ESL web components"

File: pages/views/examples/menu.njk [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: content
 title: Menu
+seoTitle: Simple menu ui example based on ESL web components
```
Comment (fshovchko) [inline pages/views/examples/menu.njk:4]: According to other seoTitles "ui" should be in uppercase like "UI"

File: pages/views/examples/scrollbar.njk [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: content
 title: Scrollbar
+seoTitle: Custom scroll ui examples based on ESL web components
```
Comment (fshovchko) [inline pages/views/examples/scrollbar.njk:4]: According to other seoTitles "ui" should be in uppercase like "UI"

File: pages/views/index.njk [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
 layout: landing
+description: Open-source UI component library that helps developers create modern-looking, effective, and user-friendly interfaces faster.
```
Comment (fshovchko) [inline pages/views/index.njk:3]: pages/views/_includes/landing/marquee.njk: 7
"Open-source UI components library that helps developers create modern-looking, effective, and user-friendly interfaces faster."
The issue word "components" is written in plural form in on case and in single form in other case

File: pages/views/utils/index.njk [hunk 10]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: collection-grid
 title: Utils
+seoTitle: Utility list of Exadel Smart Library (ESL)
```
Comment (fshovchko) [inline pages/views/utils/index.njk:4]: Utility can be written in plural form: "Utilities"

File: pages/views/introduction/index.njk [hunk 11]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: content
 title: Installation Guide
+seoTitle: ESL Installation Guide and Configuration
+description: Get started with installing Exadel Smart Library on the project. Configuring and using ESL components.
```
Comment (fshovchko) [inline pages/views/introduction/index.njk:5]: Configuring and using ESL components can be rewritten like : "Configuration and usage of ESL components"

**Pull request merged**